build/m4/aircrack_ng_pcre2.m4

```diff
@@ -0,0 +1,61 @@
+dnl Aircrack-ng
+dnl
+dnl Copyright (C) 2017 Joseph Benden <joe@benden.us>
```

Comment:
Don't forget to adjust the copyrights

Reply:
Right, should I update the date only or put my name there also?

Reply:
fixed